src/LibObjectFile.Tests/Elf/ElfSimpleTests.cs

```diff
@@ -12,6 +12,16 @@ namespace LibObjectFile.Tests.Elf
 {
     public class ElfSimpleTests : ElfTestBase
     {
+        [Test]
+        public void TryReadThrows()
+        {
+            using var stream = File.OpenRead("TestFiles/cmnlib.b00");
+            Assert.False(ElfObjectFile.TryRead(stream, out var elf, out var diagnostics));
+            Assert.NotNull(elf);
+            Assert.AreEqual(1, diagnostics.Messages.Count);
+            Assert.AreEqual(116, diagnostics.Messages[0].Id);
```

Comment:
Why not using the enum DiagnosticId directly?

Reply:
Sorry I missed that